src/main/java/org/overture/ego/model/exceptions/DontSetIdInCreateRequestException.java

```diff
@@ -0,0 +1,9 @@
+package org.overture.ego.model.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value= HttpStatus.BAD_REQUEST, reason=DontSetIdInCreateRequestException.reason)
+public class DontSetIdInCreateRequestException extends RuntimeException {
```

Comment:
can we rename this to `PostWithIdentifier`

Reply:
Sure! :-)